lib/rules/value-no-vendor-prefix/index.mjs

```diff
@@ -48,17 +45,21 @@ const rule = (primary, secondaryOptions) => {
 		}
 
 		/**
-		 * @type {{ raw: IgnoreValues, unprefixed: IgnoreValues }}
+		 * @typedef {string | RegExp} IgnoreValue
+		 * @type {{ raw: IgnoreValue[], unprefixed: IgnoreValue[] }}
 		 * @todo use Object.groupBy once Node v20 support is dropped
 		 */
 		const groups = { raw: [], unprefixed: [] };
-
-		secondaryOptions?.ignoreValues?.forEach((value) => {
+		const ignoreValues = secondaryOptions?.ignoreValues;
+		const fillGroups = (/** @type {IgnoreValue} */ value) => {
 			const useRawValue = isRegExp(value) || (value.startsWith('/') && value.match(/\/i?$/));
```

Comment:
[note] Out-of-scope. We want to unify the similar logic with another opportunity.

https://github.com/stylelint/stylelint/blob/84f13fea65658a0ec6916567086ae73e8f77af00/lib/utils/matchesStringOrRegExp.mjs#L74-L78